udata/tests/api/test_dataservices_api.py

```diff
@@ -431,9 +431,7 @@ def test_dataservice_api_create_with_unkwown_license(self):
             },
         )
         self.assert400(response)
-        self.assertEqual(
-            response.json["errors"]["license"], ["Unknown reference 'unwkown-license'"]
-        )
+        self.assertEqual(response.json["errors"]["license"], "Unknown reference 'unwkown-license'")
```

Comment:
We used to have both `list[str]` and `str` in errors fields?
Do we now have `str` only everwhere?